designdocs/user-handles.md

```diff
@@ -0,0 +1,326 @@
+# Frequency User Handles Registry
+
+## Context and Scope
+
+The Frequency blockchain aims to provide a mechanism to register and retrieve user handles on chain to make it easier to use MSA Ids.
+To achieve this, we propose the creation of a user handle ```registry``` system on frequency chain, coupled with ```MessageSourceAccount```, that allows users to choose a handle and select a suffix from a set of options allowed by the chain that together are unique.
```

Comment:
```suggestion
To achieve this, we propose the creation of a user handle ```registry``` system on the Frequency chain, coupled with ```MessageSourceAccount```, that allows users to choose a handle and select a suffix from a set of options allowed by the chain that together are unique.
```

Reply:
Or.. 
`To achieve this, we propose the creation of a user handle ```registry``` system on chain`

Reply:
Style guide note. Just "Frequency", not "the" Frequency.

Reply:
:+1: